Userland/Libraries/LibGfx/BMPLoader.cpp

```diff
@@ -1434,7 +1416,7 @@ IntSize BMPImageDecoderPlugin::size()
     if (m_context->state == BMPLoadingContext::State::Error)
         return {};
 
-    if (m_context->state < BMPLoadingContext::State::DIBDecoded && !decode_bmp_dib(*m_context))
+    if (m_context->state < BMPLoadingContext::State::DIBDecoded && decode_bmp_dib(*m_context).is_error())
```

Comment:
Same note here about manual error checking / hiding the underlying error

Reply:
Do I understand correctly that you want me to change the return type of `ImageDecoderPlugin::size` and all derived types to `ErrorOr<IntSize>` ?

Reply:
Ah I see, yeah let's not do that for now. We *should* change that to `ErrorOr<IntSize>` eventually, but we don't need to do that here.